oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+server-tools-file https://github.com/akretion/server-tools.git 8-attachm-meta
```

Comment:
I saw that the module `attachment_metadata` is dependency just to add a fields. (Fix me if I'm wrong.)
What about use directly the original view and remove this dependency to allow choose each user install optionally `attachment_metadata`?